book/website/reproducible-research/rdm/rdm-article.md

```diff
@@ -0,0 +1,61 @@
+(rr-rdm-article)=
+# Sharing your data through a Data Article
+A Data Article (also known as a Data Paper/Note/Release, or Database article) is a publication that is focused on the description of a dataset. It makes use of the traditional journal article structure, but focuses on the data collection and methodological aspects and generally not on the interpretation or discussion of the results.
+
+(rr-rdm-article-motivations)=
+##  Motivations For Sharing Data through a Data Article
+
+Data articles are in line with the {ref}`FAIR principles<rr-rdm-fair>`, especially since most publishers will encourage you to share the data through a data repository. The benefit of a Data Article is that your output will be {ref}`peer reviewed<cm-peer-review>`, something which is generally not the case for datasets that are archived on {ref}`data repositories<rr-rdm-sharing>`. It also facilitates recognition for datasets through research assessment procedures that are more traditionally focused on publication output. Publishing a data paper will therefore increase the visibility, credibility and usability of the data.
+
+
+(rr-rdm-article-options)=
+## Options to publish a Data Article
+Below you can find some journals that publish data articles or papers. The costs information was collected in February 2022. 
```

Comment:
```suggestion
Below you can find some journals that publish data articles or papers. The costs information was collected in February 2022. 

If you are planning to publish an analysis of your published data, it may be worth checking with the journal that they wouldn't consider a data article on that dataset as 'prior publication'. F1000Research has put together a (non-exhaustive) list of journals about this [here](https://f1000research.com/data-policies?utm_source=CPB&utm_medium=cms&utm_campaign=JPF15157).
```